protocols/ping/src/handler.rs

```diff
@@ -295,7 +295,7 @@ impl ConnectionHandler for Handler {
                         break;
                     }
                     Poll::Ready(Ok((stream, rtt))) => {
-                        log::debug!("latency to {} is {}ms", self.peer, rtt.as_millis());
+                        tracing::debug!("latency to {} is {}ms", self.peer, rtt.as_millis());
```

Comment:
Statements like this should use structured logs by capturing the RTT as a field instead of embedding it into the message: https://docs.rs/tracing/0.1.38/tracing/#recording-fields

Reply:
updated to this:
` tracing::event!(tracing::Level::TRACE, peer = %self.peer, ms = rtt.as_millis());`
so that we capture peer and rtt as a field. Let me know if you wanted something different here

Reply:
Yep, that makes sense. You can continue to use `tracing::debug!` though, no need to for the verbose `event!` macro. We can change the message to something like: `Successfully pinged peer`.

The `ms` field should probably be called `rtt_ms` or something.

And with the newly introduced span inside `Connection`, I don't think we need to log the `PeerId` anymore right? It should show up automatically now :)